include/solvers/multiphysics_interface.h

```diff
@@ -77,55 +75,149 @@ class MultiphysicsInterface
   {
     if (physics_id == PhysicsID::heat_transfer)
       {
-        pcout << "--------------" << std::endl
-              << "Heat Transfer" << std::endl
-              << "--------------" << std::endl;
+        pcout << "Heat Transfer" << std::endl << "--------------" << std::endl;
       }
     else if (physics_id == PhysicsID::tracer)
       {
-        pcout << "-------" << std::endl
-              << "Tracer" << std::endl
-              << "-------" << std::endl;
+        pcout << "Tracer" << std::endl << "-------" << std::endl;
```

Comment:
I think keeping the message as it was before is a better idea. It separates the message from the upper line in the log.

Reply:
It is still the case! Now it is handled by the pre_solve and post_solve functions, because otherwise it was ugly for presolved physics. In the end, we still have a line before the announce message, no worries!